### ISSUE TITLE
MO2 v2.4 API Change Breaks Plugin:  AttributeError: 'IPluginGame' object has no attribute 'savegameExtension'

### ISSUE THREAD

**Issue by ron-courtright** [AnyOldName3/modorganizer-orphaned_script_extender_save_deleter#3]
Here is the log traceback:

[2021-02-23 22:57:28.792 E] [error.cpp:24] Traceback (most recent call last):
[2021-02-23 22:57:28.792 E] [error.cpp:24]   File "C:/Games/Modding/MO2/plugins/OrphanedScriptExtenderSaveDeleter.py", line 57, in display
[2021-02-23 22:57:28.792 E] [error.cpp:24]     gameSaveExtension = managedGame.savegameExtension()
[2021-02-23 22:57:28.793 E] [error.cpp:24] AttributeError: 'IPluginGame' object has no attribute 'savegameExtension'
[2021-02-23 22:57:28.793 E] Plugin "Orphaned Script Extender Save Deleter" failed: Traceback (most recent call last):
[2021-02-23 22:57:28.793 E]   File "C:/Games/Modding/MO2/plugins/OrphanedScriptExtenderSaveDeleter.py", line 57, in display
[2021-02-23 22:57:28.793 E]     gameSaveExtension = managedGame.savegameExtension()
[2021-02-23 22:57:28.793 E] AttributeError: 'IPluginGame' object has no attribute 'savegameExtension'
[2021-02-23 22:57:28.793 E]

**Comment by AnyOldName3** [comment]
Either use the version from Holt's pull request here https://github.com/AnyOldName3/modorganizer-orphaned_script_extender_save_deleter/pull/2 or just stop using the plugin. SKSE64 has fixed the bug, so you shouldn't have orphaned saves any more.

**Comment by ron-courtright** [comment]
I took the change and it worked for me.  So great.  Thank you and Holt for the support.

That said, since I am using the latest SKSE build and if SKSE64 was fixed why am I having this problem with orphaned saves?

ETA:  Hmm, my bad.  Yes, of course SKSE removes the orphan save nowadays.  I just proved it to myself.  So the utility of this script is that Mod Organizer 2 does NOT remove the orphaned file after I delete a save from the Save file list.  Or rather, if I delete a number of accumulated non-contiguous auto saves and/or Quick saves.

And I again offer my thanks to all who made this utility exist.
